api/oci.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package api;
+option go_package = "github.com/bitnami-labs/charts-syncer/api;api";
+
+
+message OciIndex {
```

Comment:
I'd add a comment here explaining that this is a proposed custom index

Reply:
+1